### PR TITLE
MVP Opera support

### PR DESCRIPTION
**Pull request by owocki**
##### Description

MVP of Opera ssupport - tested on Opera + Chrome

##### Refers/Fixes

https://github.com/gitcoinco/web/issues/5317#issuecomment-547836980

##### Testing

Tested on Opera + Chrome

If approahc makes  sense I will do one last batch of testing and merge

### CODE REVIEW

**Comment by owocki** [comment]
this one is for you @yorkerhodes3 !

File: app/assets/v2/js/shared.js [hunk 1]
```diff
@@ -705,10 +705,12 @@ var currentNetwork = function(network) {
           $('#current-network').text(gettext('Metamask Locked'));
           $('#navbar-network-banner').html(info);
         } else {
-          info = gettext('Metamask not connected. ') +
-            '<button id="metamask_connect" onclick="approve_metamask()">Click here to connect to metamask</button>';
-          $('#current-network').text(gettext('Metamask Not Connected'));
-          $('#navbar-network-banner').html(info);
+          if (window.ethereum._metamask) {
```
Comment (thelostone-mc) [inline app/assets/v2/js/shared.js]: can we make this into an `else if` in `L107` ?

File: app/assets/v2/js/shared.js [hunk 2]
```diff
@@ -746,10 +748,12 @@ var currentNetwork = function(network) {
...
```
Comment (thelostone-mc) [inline app/assets/v2/js/shared.js]: can we make this into an `else if` in `L749`?

**Comment by thelostone-mc** [comment]
approach looks alright to me !  If we can get one more ronud of testing in + address those comments -> we can get this in this week ❤️

File: app/assets/v2/js/shared.js (hunk 2 above)
Comment (owocki) [reply app/assets/v2/js/shared.js]: cf5de1e0c91ed3befd9c8967589360048b7cba4c

File: app/assets/v2/js/shared.js (hunk 1 above)
Comment (owocki) [reply app/assets/v2/js/shared.js]: cf5de1e0c91ed3befd9c8967589360048b7cba4c

**Comment by owocki** [comment]
just regression tested kudos send/ grants creation/tip sending/bounty creation on metamask and all went fine.. can i have a review @danlipert @thelostone-mc @octavioamu

**Comment by octavioamu** [comment]
Im trying to send  a tip but I got stuck here, Is something more I need to do? 
image
image
when I played with this I remember the problem in that was 
```
web3.eth.sendTransaction({
            to: destinationAccount,
            value: amountInDenom,
```
since `from:` is missing

**Comment by octavioamu** [comment]
Yes I added the from and now is working I will commit @owocki 

image

**Comment by owocki** [comment]
thanks @octavioamu - can u push it back to my branch??

**Comment by octavioamu** [comment]
Just tested kudos and tips working perfectly. Going to test a little more, excited with this

**Comment by owocki** [comment]
@octavioamu thanks for f631fb5a91daf80a3c74f832ce71bc866ac664fb ; im concerned that there might be other places in the codebase where we implicitly expect `from:` to exist .. thereby peoples first opera experiences with gitcoin may fail.. 

since i wasnt able to reproduce the issue.. im not sure how to test for that tho.. what opera version r u on?

**Comment by octavioamu** [comment]
Im with the last, tested with the desktop browser. Yes I think we can add the other places in a  progressive  way.

**Pull request merged**